notebooks/templates/Project.toml

```diff
@@ -0,0 +1,4 @@
+[deps]
+DS-Julia2925 = "d4d1b526-0a71-4e37-87a1-4602eac14fc5"
+DSJulia = "36755c37-737e-4b39-b6ef-e2f953844095"
+TeachingTools = "52a36ae6-b042-44a7-b1ae-8e210fe21337"
```

Comment:
Is this a package somewhere?

Reply:
`TeachingTools`

Reply:
Ah, oopsie that is from a previous attempt at making a package ;) I will remove this.

Reply:
Actually the entire Project.toml should not be there